Api driven logan client updates (#1370)
tinacms/tinacms.org: content/docs/cli-overview.md

| Argument        | Description                                                                                                                                                             |
| --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `-c`            | `-c` allows you to run a command as a child process. For example, you could run your next project alongside the graphQL server `yarn tinacms server:start -c next dev`. |
| `--port <port>` | Specify a port to run the server on. (default 4001)                                                                                                                     |
| `--noWatch`     | Don't regenerate config on file changes. This option is good to add when building in CI or if you do not want to watch the file system for changes.                     |
| `--noSDK`       | Don't generate the generated client SDK. [Read here](/docs/graphql/client/) for more details about the SDK                                                              |

### `tinacms build`

> This command compiles and checks schema, client and types. As well it will check your content to make sure it has finished indexing.

Reviewer: ```suggestion
This command compiles and validates the schema and generates the client and types. It will also ensure your content has finished indexing.
```
